paymentsheet/src/main/java/com/stripe/android/paymentsheet/ui/DefaultPaymentMethodLabel.kt

```diff
@@ -0,0 +1,41 @@
+package com.stripe.android.paymentsheet.ui
+
+import android.content.res.Configuration
+import androidx.annotation.RestrictTo
+import androidx.compose.material.MaterialTheme
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.text.style.TextOverflow
+import androidx.compose.ui.tooling.preview.Preview
+import com.stripe.android.paymentsheet.R
+import com.stripe.android.uicore.preview.DefaultStripeThemePreviewRow
+import com.stripe.android.uicore.stripeColors
+
+@RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
```

Comment:
you shouldn't need to include the RestrictTo annotations if the type is internal. 

RestrictTo ensures public functions don't show up in our public API if we don't want them to. In this case, since the function is internal, it wouldn't show up in our public API anyways.